local/gugcat/index.php

```diff
@@ -303,9 +303,11 @@
         $scaleid = $activity->gradeitem->scaleid;
         $gradetype = $activity->gradeitem->gradetype;
         $grademax = $activity->gradeitem->grademax;
-        if($gradetype != GRADE_TYPE_VALUE && $gradetype != GRADE_TYPE_SCALE){
-            $importerror[] = $activity->gradeitemid;
-        }else if($gradetype == GRADE_TYPE_SCALE && !local_gugcat::is_scheduleAscale($gradetype, $grademax)){
+        $grademin = $selectedmodule->gradeitem->grademin;
```

Comment:
I think we need to check the $activity and not the $selectedmodule for $grademin.